gpkit/solution_array.py

```diff
@@ -29,16 +34,27 @@ class SolutionArray(DictOfLists):
     >>> x = gpkit.Variable("x")
     >>> x_min = gpkit.Variable("x_{min}", 2)
     >>> sol = gpkit.Model(x, [x >= x_min]).solve(verbosity=0)
-    >>> # ACCESS VALUES
+    >>>
+    >>> # VALUES
     >>> values = [sol(x), sol.subinto(x), sol["variables"]["x"]]
     >>> assert all(np.array(values) == 2)
-    >>> # ACCESS SENSITIVITIES
+    >>>
+    >>> # SENSITIVITIES
     >>> senss = [sol.sens(x_min), sol.senssubinto(x_min)]
     >>> senss.append(sol["sensitivities"]["variables"]["x_{min}"])
     >>> assert all(np.array(senss) == 1)
 
     """
 
+    def append_parse(self, *args, **kwargs):
+        """Parses and then appends a GP-like result.
+
+        Arguments
+        ---------
+        *args, **kwargs : passed to parse_result()
+        """
+        self.append(parse_result(*args, **kwargs))
+
```

Comment:
um, is this better than just calling the commands in series? It's certainly more obfuscated.